<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="OrderFrame">
     <property name="styleSheet">
      <string notr="true">QFrame {
	background-color: #31363b;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="TotalLabel">
        <property name="font">
         <font>
          <family>Futura</family>
          <pointsize>50</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
    color: #ffff00;
    background-color: #31363b; 
    border-color: #ffff00;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
        </property>
        <property name="text">
         <string>Rp 0.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="SessionLabel">
          <property name="styleSheet">
           <string notr="true">QLabel{
    color: #ffff00;
    background-color: #31363b; 
    border-color: #ffff00;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="OperatorLabel">
          <property name="styleSheet">
           <string notr="true">QLabel{
    color: #ffff00;
    background-color: #31363b; 
    border-color: #ffff00;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="CommandLabel">
          <property name="styleSheet">
           <string notr="true">QLabel{
    color: #ffff00;
    background-color: #31363b; 
    border-color: #ffff00;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="ProductFrame">
              <property name="styleSheet">
               <string notr="true">QFrame#ProductFrame{
	background-color:#ffff00;
    border-color:#ffff00;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1">
               <property name="leftMargin">
                <number>2</number>
               </property>
               <property name="topMargin">
                <number>2</number>
               </property>
               <property name="rightMargin">
                <number>2</number>
               </property>
               <property name="bottomMargin">
                <number>2</number>
               </property>
               <item>
                <widget class="QLabel" name="ProductLabel">
                 <property name="font">
                  <font>
                   <family>Futura</family>
                   <pointsize>25</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel {
    background-color:#ffff00;
	color: #000000;
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="TotalFrame">
              <property name="styleSheet">
               <string notr="true">QFrame#TotalFrame{
	background-color:#ffff00;
    border-color:#ffff00;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,0">
               <property name="leftMargin">
                <number>2</number>
               </property>
               <property name="topMargin">
                <number>2</number>
               </property>
               <property name="rightMargin">
                <number>2</number>
               </property>
               <property name="bottomMargin">
                <number>2</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="PriceLabel">
                   <property name="font">
                    <font>
                     <family>Futura</family>
                     <pointsize>18</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLabel {
    background-color:#ffff00;
	color: #000000;
}</string>
                   </property>
                   <property name="text">
                    <string>@</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="QuantityButton">
                   <property name="styleSheet">
                    <string notr="true">QPushButton{
    color: #ffff00;
    background-color: #31363b;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #31363b;
	background-color:#ffff00;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
                   </property>
                   <property name="text">
                    <string>0 Pcs</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="LineTotalLabel">
                 <property name="minimumSize">
                  <size>
                   <width>75</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Futura</family>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel {
    background-color:#ffff00;
	color: #000000;
}</string>
                 </property>
                 <property name="text">
                  <string>Rp 0.0</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QFrame" name="CommandFrame">
          <property name="styleSheet">
           <string notr="true">QFrame#CommandFrame{
	background-color:#ffff00;
    border-color:#ffff00;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,1,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
             <property name="spacing">
              <number>13</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="LineButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Futura</family>
                 <pointsize>13</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
    color: #31363b;
    background-color: #ffff00;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #ffff00;
	background-color:#31363b;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
               </property>
               <property name="text">
                <string>Line</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="ProductButton">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Futura</family>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
    color: #31363b;
    background-color: #ffff00;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #ffff00;
	background-color:#31363b;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
               </property>
               <property name="text">
                <string>Product</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QPushButton" name="PaymentButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Futura</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
    color: #31363b;
    background-color: #ffff00;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #ffff00;
	background-color:#31363b;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
               </property>
               <property name="text">
                <string>Payment</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>13</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="HoldButton">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
    color: #31363b;
    background-color: #ffff00;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #ffff00;
	background-color:#31363b;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
               </property>
               <property name="text">
                <string>Hold</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="ExitButton">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Futura</family>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
    color: #31363b;
    background-color: #ffff00;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #ffff00;
	background-color:#31363b;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
               </property>
               <property name="text">
                <string>Exit</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
